cmake/BlocksDSi.cmake

```diff
@@ -0,0 +1,5 @@
+cmake_minimum_required(VERSION 3.16)
+
+set(NDS_DSI_EXCLUSIVE ON)
+
+include(${CMAKE_CURRENT_LIST_DIR}/BlocksDS.cmake)
```

Comment:
Huh, is `cmake/Platform/NintendoDS.cmake` supposed to be empty? Maybe add a comment there to make it clear.

Reply:
Actually not, I had misunderstood which variables got set when. Apparently the `_INIT` flags and flags regarding the executable should be set in a Platform file. Fixed now.